mParticle-Apple-SDK/mParticle.m

```diff
@@ -438,6 +438,13 @@ - (NSString *)uniqueIdentifier {
     return [MParticle sharedInstance].stateMachine.consumerInfo.uniqueIdentifier;
 }
 
+- (void)setUploadInterval:(NSTimeInterval)uploadInterval {
+    if (uploadInterval && uploadInterval != self.backendController.uploadInterval) {
```

Comment:
```suggestion
    if (uploadInterval >= 1.0 && uploadInterval != self.backendController.uploadInterval) {
```

You mention that the minimum upload interval is 1 second, but this code allows any interval except for 0.0 seconds, including negative intervals.

Reply:
The setter for the uploadInterval in MPBackendController already enforced this but I guess it doesn't hurt to do it here too. https://github.com/mParticle/mparticle-apple-sdk/blob/81099982699f998d4f26f370b05cf06023775737/mParticle-Apple-SDK/MPBackendController.mm#L1115